aiohttp/web_response.py

```diff
@@ -176,7 +176,7 @@ def enable_compression(
     ) -> None:
         """Enables response compression encoding."""
         # Backwards compatibility for when force was a bool <0.17.
-        if type(force) == bool:
+        if isinstance(force, bool):
```

Comment:
Are integers boolean? I remember there were some corner cases a around this. Or perhaps this was done with performance in mind.. FWIW this is not just a formatting change. It checks for subclasses too. Is this necessary?

Reply:
> Are integers boolean

No, other way round. bool is subclass of int, so this shoudn't make any difference. It's also ancient code that could be removed at this point anyway..